models/RwdImage.php

```diff
@@ -255,6 +277,25 @@ public function background( $selector, $size ) {
 					$rwd_background_styles[ $media ] = array();
 				}
 
+				// get retina sources.
+				if ( $option->retina_options ) {
+					// get bg media size.
+					preg_match( '/\((.+)\)/', $option->bg, $bg_media_size );
```

Comment:
it's not clear what you get here. trying to get something in brackets can be very buggy. it's better to specify better regular expression